Corona-Warn-App/src/main/java/de/rki/coronawarnapp/notification/NotificationHelper.kt

```diff
@@ -94,6 +94,11 @@ object NotificationHelper {
         Timber.v("Canceled future notifications with id: %s", notificationId)
     }
 
+    fun cancelCurrentNotification(notificationId: Int) {
+        NotificationManagerCompat.from(CoronaWarnApplication.getAppContext()).cancel(notificationId)
```

Comment:
Has a check to be added if the notification exists are is this handled by the `NotificationManagerCompat`?

Reply:
Is handled by the manager.